jaxrs/src/main/java/org/jboss/as/jaxrs/JaxrsSubsystemDefinition.java

```diff
@@ -63,6 +64,13 @@ public class JaxrsSubsystemDefinition extends SimpleResourceDefinition {
                  .setRemoveHandler(ReloadRequiredRemoveStepHandler.INSTANCE));
     }
 
+    @Override
+    public void registerAttributes(ManagementResourceRegistration registration) {
+       for (AttributeDefinition definition : JaxrsAttribute.ATTRIBUTES) {
+          registration.registerReadWriteAttribute(definition, null, new JaxrsParamHandler(definition));
```

Comment:
FYI re https://issues.redhat.com/browse/WFLY-18257. I'm not requesting a change; just FYI.

Reply:
That's worthwhile change to make now (though, eventually, these attribute definition parameters will be made obsolete by https://github.com/wildfly/wildfly-core/pull/5563).